modules/fluid_properties/doc/content/source/fluidproperties/SalineMoltenSaltFluidProperties.md

```diff
@@ -0,0 +1,42 @@
+# SalineMoltenSaltFluidProperties
+
+!syntax description /FluidProperties/SalineMoltenSaltFluidProperties
+
+The mole fractions of the various elements / molecules present in the salt must sum to one.
+The following properties can be computed:
+
+- density
+- isobaric specific heat
+- enthalpy
+- viscosity
+- thermal diffusivity
+
+!alert note
+Saline must be installed alongside MOOSE using the `contrib/saline` submodule in the fluid properties
+module to be able to use this fluid property object. To initialize/download the saline submodule:
+
+```bash
+git submodule update --init contrib/saline
+```
+
+## Example input syntax
+
+In this example, the fluid properties for a `LiF-NaF-KF` salt mixture are computed by a
+`SalineMoltenSaltFluidProperties`.
+
+!listing modules/fluid_properties/test/tests/saline/test.i block=FluidProperties
+
+It leverages the salt property input data in the `saline_custom.prp` file below.
+The data is sorten in columns. The first column is the name of the salt, then its id, then
+a number of thermo-physical properties with additional metadata such as the range of validity,
+the uncertainty and a short name for the reference publication providing the data.
```

Comment:
This explanation still does not give enough information for someone to make their own prp. The example is very useful, but there are a lot of columns that I still don't understand. Is there documentation on this file format somewhere else that we can link to? That would be fine; otherwise, I think we need to spell out the requirements for the format in greater detail, like a table with column number to the description of what goes there (with units). Also, is there always just a single row in this file? How do those reference strings get used?

Reply:
all good questions. i think we ll want to reference somewhere else here